service/static/index.html

```diff
@@ -0,0 +1,108 @@
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="static/css/blue_bootstrap.min.css">
+    <script type="text/javascript" src = "static/js/jquery-3.1.1.min.js"></script>
+    <script type="text/javascript" src="static/js/rest_api.js"></script>
+  </head>
+  <body>
+    <div class="container">
+        <div class="page-header">
+            <h1>Wishlist REST API Service</h1>
+        </div>
+
+        <!-- Flash Message -->
+        <div class="table-responsive">
+          <table class="table">
+            <tr><td>Status:</td><td><span id="flash_message"></span></td></tr>
+          </table>
+        </div>
+
+        <div class="col-md-12" id="form_data">
+          Create, Retrieve, Update, and Delete a Wishlist:
+          <div class="well">
+            <div class="form-horizontal">
+              <div class="form-group">
+                <label class="control-label col-sm-2" for="wishlist_id">Wishlist ID:</label>
+                <div class="col-sm-6">
+                  <input type="text" class="form-control" id="wishlist_id" placeholder="Enter ID of Wishlist">
+                </div>
+                <div class="col-sm-4">
+                  <button type="submit" class="btn btn-primary" id="retrieve-btn">Retrieve</button>
+                  <button type="submit" class="btn btn-danger" id="delete-btn">Delete</button>
+                  &nbsp;&nbsp;&nbsp;
+                  <a href="items.html">
+                    <button>Item Page</button>
+                  </a>
+                </div>
+              </div>
+            </div> <!-- form div -->
+
+            <div class="form-horizontal">
+              <div class="form-group">
+                <label class="control-label col-sm-2" for="wishlist_name">Name:</label>
+                <div class="col-sm-10">
+                  <input type="text" class="form-control" id="wishlist_name" placeholder="Enter name for Wishlist">
+                </div>
+              </div>
+              <div class="form-group">
+                <label class="control-label col-sm-2" for="wishlist_user_id">User ID:</label>
+                <div class="col-sm-10">
+                  <input type="text" class="form-control" id="wishlist_user_id" placeholder="Enter user id for Wishlist">
+                </div>
+              </div>
+              <div class="form-group">
+                <label class="control-label col-sm-2" for="wishlist_items">Items:</label>
+                <div class="col-sm-10">
+                  <input type="text" class="form-control" id="wishlist_items" placeholder="Enter items for Wishlist in format [item1, item2, ...]">
+                </div>
+              </div>
+              <div class="form-group">
+                <label class="control-label col-sm-2" for="wishlist_status">Status:</label>
+                <div class="col-sm-10">
+                  <select class="form-control" id="wishlist_status">
```

Comment:
We can have the status as enabled or disabled.

Reply:
Done